src/style.css

```diff
@@ -1,14 +1,60 @@
-@import "tailwindcss/base";
-@import "tailwindcss/components";
-@import "tailwindcss/utilities";
+@import "tailwindcss" source(".");
+@plugin "@tailwindcss/typography";
+
+@plugin "daisyui";
+@plugin "daisyui/theme" {
+  name: "halloween";
+  default: true;
+  --color-primary: var(--color-green-600);
+  --color-secondary: var(--color-cyan-500);
+}
+
+@theme {
+  --font-*: initial;
+  --font-sans: Inter, ui-sans-serif, system-ui, sans-serif, "Apple Color Emoji",
+    "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+}
+
+/*
+  The default border color has changed to `currentcolor` in Tailwind CSS v4,
+  so we've added these compatibility styles to make sure everything still
+  looks the same as it did with Tailwind CSS v3.
+
+  If we ever want to remove these styles, we need to add an explicit border
+  color utility to any element that depends on these defaults.
+*/
+@layer base {
+  *,
+  ::after,
+  ::before,
+  ::backdrop,
+  ::file-selector-button {
+    border-color: var(--color-gray-200, currentcolor);
+  }
+}
+
+.lemmy-select {
+  @apply select select-sm w-fit text-gray-400;
+}
+
+.text-gradient {
+  @apply bg-linear-to-r bg-clip-text text-transparent from-green-400 to-green-600;
+}
+
+.card-gradient {
+  @apply bg-linear-to-r from-zinc-500/5 via-cyan-600/15 to-zinc-500/5;
```

Comment:
Tailwind v4 is smart about calculating opacity now and doesn't need brackets to use values other than 100%, 75%, and 50% anymore.